buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -91,7 +91,7 @@ SPECS = ['ppc64_aix'      : CURRENT_RELEASES,
 
 // SHORT_NAMES is used for PullRequest triggers
 // TODO Combine SHORT_NAMES and SPECS
-SHORT_NAMES = ['all' : 'all',
+SHORT_NAMES = ['all' : 'ppc64le_linux,s390x_linux,x86-64_linux,x86-64_linux_xl,ppc64_aix,x86-64_windows,x86-32_windows,x86-64_mac',
```

Comment:
Could this be a List instead of a comma separated strings (thus remove `LONG_PLATFORM.tokenize(',')` in pipeline-functions)?